flink connector databend can't read

```
link SQL> CREATE TABLE dproducts (id INT,name String,description String, PRIMARY KEY (`id`) NOT ENFORCED)
> WITH ('connector' = 'databend',
> 'url'='databend://localhost:8000',
> 'username'='wubx',
> 'password'='wubxwubx',
> 'database-name'='default',
> 'table-name'='dproducts',
> 'sink.batch-size' = '5',
> 'sink.ignore-delete' = 'false',
> 'sink.flush-interval' = '1000',
> 'sink.max-retries' = '3');
[INFO] Execute statement succeed.

Flink SQL> select * from dproducts;
[ERROR] Could not execute SQL statement. Reason:
java.io.NotSerializableException: org.apache.flink.connector.databend.internal.converter.DatabendRowConverter

```

--- a/src/main/java/org/apache/flink/connector/databend/internal/converter/DatabendRowConverter.java
+++ b/src/main/java/org/apache/flink/connector/databend/internal/converter/DatabendRowConverter.java
@@ -1,30 +1,23 @@
 package org.apache.flink.connector.databend.internal.converter;
 
-import static org.apache.flink.connector.databend.internal.converter.DatabendConverterUtils.BOOL_TRUE;
-import static org.apache.flink.connector.databend.util.DatabendUtil.toEpochDayOneTimestamp;
-
 import com.databend.jdbc.DatabendPreparedStatement;
 import com.databend.jdbc.DatabendResultSet;
+import org.apache.flink.table.data.*;
+import org.apache.flink.table.types.logical.*;
+import org.apache.flink.table.types.logical.RowType.RowField;
+import org.apache.flink.util.Preconditions;
+
 import java.io.Serializable;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.sql.*;
 import java.time.LocalDate;
 import java.time.LocalTime;
-import org.apache.flink.table.data.DecimalData;
-import org.apache.flink.table.data.GenericRowData;
-import org.apache.flink.table.data.RowData;
-import org.apache.flink.table.data.StringData;
-import org.apache.flink.table.data.TimestampData;
-import org.apache.flink.table.types.logical.DecimalType;
-import org.apache.flink.table.types.logical.LocalZonedTimestampType;
-import org.apache.flink.table.types.logical.LogicalType;
-import org.apache.flink.table.types.logical.RowType;
-import org.apache.flink.table.types.logical.RowType.RowField;
-import org.apache.flink.table.types.logical.TimestampType;
-import org.apache.flink.util.Preconditions;
 
-public class DatabendRowConverter {
+import static org.apache.flink.connector.databend.internal.converter.DatabendConverterUtils.BOOL_TRUE;
+import static org.apache.flink.connector.databend.util.DatabendUtil.toEpochDayOneTimestamp;
+
+public class DatabendRowConverter implements Serializable{
     private static final long serialVersionUID = 1L;
 
     private final RowType rowType;

